### PR TITLE
[szenikeu] Add new extractor

### PR DESCRIPTION
**Pull request by benages**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [ ] Improvement
- [x] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information
New extractor for szenik.eu. Closes https://github.com/rg3/youtube-dl/issues/12408

### CODE REVIEW

File: youtube_dl/extractor/szenikeu.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        webpage = self._download_webpage(url, video_id)
+
+        title = self._html_search_meta('og:title', webpage)
+        title = title.split(' |')[0]
+        thumbnail = self._html_search_meta('og:image', webpage)
+        description = self._html_search_meta('og:description', webpage, fatal=False)
+
+        # The folder name is obtained from the image thumbnail
+        video_folder = thumbnail.rsplit('/', 1)[1].split('.')[0]
+        baseurl = 'http://vod.szenik.eu:8134/media/szenik/%s/' % video_folder
```
Comment (dstftw) [inline youtube_dl/extractor/szenikeu.py]: This should be obtained from http://www.szenik.eu/visionneuse/visio_v7_js.php?key=bw0Y9lgV47&width=670 and current code should be used as a fallback.

File: youtube_dl/extractor/szenikeu.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            'description': 'Le Tonhalle-Orchester de Zurich et la star des percussionnistes Martin Grubinger jouent Frozen in Time d’Avner Dorman et Le Sacre du printemps de Stravinski. Un concert à revoir en intégralité sur szenik Live.',
+            'thumbnail': 'http://www.szenik.eu/videoimages/grand/Tonhalle_Frozen_in_time.jpg',
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        title = self._html_search_meta('og:title', webpage)
```
Comment (dstftw) [inline youtube_dl/extractor/szenikeu.py]: `self._og_search_title`.

File: youtube_dl/extractor/szenikeu.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        title = title.split(' |')[0]
+        thumbnail = self._html_search_meta('og:image', webpage)
+        description = self._html_search_meta('og:description', webpage, fatal=False)
```
Comment (dstftw) [inline youtube_dl/extractor/szenikeu.py]: `_og_search_description`.

File: youtube_dl/extractor/szenikeu.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        title = self._html_search_meta('og:title', webpage)
+        title = title.split(' |')[0]
+        thumbnail = self._html_search_meta('og:image', webpage)
```
Comment (dstftw) [inline youtube_dl/extractor/szenikeu.py]: `_og_search_thumbnail`.

File: youtube_dl/extractor/szenikeu.py [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+        description = self._html_search_meta('og:description', webpage, fatal=False)
+
+        # The folder name is obtained from the image thumbnail
+        video_folder = thumbnail.rsplit('/', 1)[1].split('.')[0]
+        baseurl = 'http://vod.szenik.eu:8134/media/szenik/%s/' % video_folder
+
+        m3u8_url = baseurl + 'index.m3u8'
+        formats = self._extract_m3u8_formats(
+            m3u8_url, video_id, 'mp4', entry_protocol='m3u8_native',
+            m3u8_id='hls')
```
Comment (dstftw) [inline youtube_dl/extractor/szenikeu.py]: Should be non fatal.

**Comment by benages** [comment]
I don't know how to obtain the video info from http://www.szenik.eu/visionneuse/visio_v7_js.php?key=bw0Y9lgV47&width=670.
I have made the other changes.

**Pull request closed without merging**